ui/src/app/pages/signed-in/access-renewal-notification.spec.tsx

```diff
@@ -138,104 +153,163 @@ describe('Access Renewal Notification', () => {
       AccessTierShortNames.Controlled,
       allCompleteNotExpiring,
+      EMPTY_STRING,
+      EMPTY_STRING,
     ],
     [
       true,
       'for RT when there is one expiring module',
       AccessTierShortNames.Registered,
       allCompleteOneExpiring(AccessModule.DATAUSERCODEOFCONDUCT),
+      colors.notificationBanner.EXPIRED,
+      colors.danger,
+    ],
+    [
+      true,
+      'for RT when there is one module expiring in couple of weeks',
+      AccessTierShortNames.Registered,
+      allCompleteOneExpiringInMoreThanTwoWeeks(
+        AccessModule.DATAUSERCODEOFCONDUCT
+      ),
+      colors.notificationBanner.EXPIRING_SOON,
+      colors.primary,
     ],
     [
       // Showing the RT banner appears in this case, so CT is not needed
       false,
       'for CT when there is one expiring module',
       AccessTierShortNames.Controlled,
       allCompleteOneExpiring(AccessModule.DATAUSERCODEOFCONDUCT),
+      EMPTY_STRING,
+      EMPTY_STRING,
     ],
     [
       true,
       'for RT when there is one expired module',
       AccessTierShortNames.Registered,
       allCompleteOneExpired(AccessModule.DATAUSERCODEOFCONDUCT),
+      colors.notificationBanner.EXPIRED,
+      colors.danger,
     ],
     [
       // Showing the RT banner appears in this case, so CT is not needed
       false,
       'for CT when there is one expired module',
       AccessTierShortNames.Controlled,
       allCompleteOneExpired(AccessModule.DATAUSERCODEOFCONDUCT),
+      EMPTY_STRING,
+      EMPTY_STRING,
     ],
     [
       false,
       'for RT when only CT training is expiring',
       AccessTierShortNames.Registered,
       allCompleteOneExpiring(AccessModule.CTCOMPLIANCETRAINING),
+      EMPTY_STRING,
+      EMPTY_STRING,
     ],
     [
       false,
       'for RT when only CT training is expired',
       AccessTierShortNames.Registered,
       allCompleteOneExpired(AccessModule.CTCOMPLIANCETRAINING),
+      EMPTY_STRING,
+      EMPTY_STRING,
     ],
     [
       true,
       'for CT when only CT training is expiring',
       AccessTierShortNames.Controlled,
       allCompleteOneExpiring(AccessModule.CTCOMPLIANCETRAINING),
+      colors.notificationBanner.EXPIRED,
+      colors.danger,
+    ],
+    [
+      true,
+      'for CT when only CT training is expiring in couple of weeks',
+      AccessTierShortNames.Controlled,
+      allCompleteOneExpiringInMoreThanTwoWeeks(
+        AccessModule.CTCOMPLIANCETRAINING
+      ),
+      colors.notificationBanner.EXPIRING_SOON,
+      colors.primary,
     ],
     [
       true,
       'for CT when only CT training is expired',
       AccessTierShortNames.Controlled,
       allCompleteOneExpired(AccessModule.CTCOMPLIANCETRAINING),
+      colors.notificationBanner.EXPIRED,
+      colors.danger,
     ],
     [
       true,
       'for RT when RT is expiring sooner',
       AccessTierShortNames.Registered,
       rtExpiresFirst,
+      colors.notificationBanner.EXPIRED,
```

Comment:
this does not look correct

Reply:
The variable name is not correct, it should not be expired but something like EXPIRING_IN_A_WEEK or something

Reply:
ahhhhh ok thanks

Reply:
then these tests look good